app/assets/javascripts/order.js

```diff
@@ -4,8 +4,12 @@ $(document).ready(function() {
   $("#new_order").submit(function(event) {
     var $form = $(this);
     $form.find("button").prop("disabled", true);
-    Stripe.card.createToken($form, stripeResponseHandler);
-    return false;
+    if $("card_number").length;
+      Stripe.card.createToken($form, stripeResponseHandler);
+      return false;
+    else {
```

Comment:
Unreachable 'else' after 'return'.<br>Expected an identifier and instead saw 'else'.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.